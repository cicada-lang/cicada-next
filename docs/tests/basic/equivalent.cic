compute equivalent Trivial {
  sole
}

compute equivalent Trivial {
  sole
  | refl
  = sole
}

compute equivalent Trivial {
  sole
  | refl
  = sole
  | refl
  = sole
}

// NOTE via `refl` by default

compute equivalent Trivial {
  sole
}

compute equivalent Trivial {
  sole
  = sole
}

compute equivalent Trivial {
  sole
  = sole
  = sole
}
