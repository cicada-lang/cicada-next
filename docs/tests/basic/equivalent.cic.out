refl(implicit Trivial, implicit sole): Equal(Trivial, sole, sole)
refl(implicit Trivial, implicit sole): Equal(Trivial, sole, sole)
refl(implicit Trivial, implicit sole): Equal(Trivial, sole, sole)
refl(implicit Trivial, implicit sole): Equal(Trivial, sole, sole)
refl(implicit Trivial, implicit sole): Equal(Trivial, sole, sole)
refl(implicit Trivial, implicit sole): Equal(Trivial, sole, sole)
